package.json

```diff
@@ -32,11 +33,14 @@
     "prettier": "^1.16.4",
     "pretty-quick": "^1.10.0",
     "serve": "^11.3.2",
+    "typescript": "^4.4.4",
     "webpack": "^5.11.0",
     "webpack-cli": "^4.2.0"
   },
   "scripts": {
     "build": "webpack",
+    "typegen": "tsc",
```

Comment:
Personally i like to keep them separate since type generation is pretty long, and is often not required when building the project. I'd rather put it in `ci` that way we know we can spot typing issues quickly and ensure we have up-to-date type definitions when publishing the package since it is called in preversion.

Reply:
Putting it in `ci` seems like a good solution to me!